tests/compatibility.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*
+ * Ensures compatibility with PHPUnit < 6.x
+ */
+namespace PHPUnit\Framework\Constraint {
+    if (!class_exists('PHPUnit\Framework\Constraint\Constraint') && class_exists('PHPUnit_Framework_Constraint')) {
+        abstract class Constraint extends \PHPUnit_Framework_Constraint {}
+    }
+}
+namespace PHPUnit\Framework {
+    if (!class_exists('PHPUnit\Framework\TestCase') && class_exists('PHPUnit_Framework_TestCase')) {
+        abstract class TestCase extends \PHPUnit_Framework_TestCase {
+            /**
+             * @param string $exception
+             */
+            public function expectException($exception)
```

Comment:
This file better to be taken from `yii2` repo - it is more complex there:
https://github.com/yiisoft/yii2/blob/master/tests/compatibility.php